src/Atata/Logging/LogManager.cs

```diff
@@ -116,10 +116,10 @@ public void Fatal(string message, Exception exception = null)
         /// <param name="section">The log section.</param>
         /// <example>This sample shows how to log the data insertion to some control in the scope of the control.
         /// <code>
-        /// string value = "new_value";
-        /// Log.Start(new DataAdditionLogSection(this, value));
+        /// string value = <c>"new_value"</c>;

```

Comment:
No need to wrap with `<c>`, as this is an example already wrapped with `<code>`.

```suggestion
        /// string value = "new_value";
```